packages/widgets/src/menu.ts

```diff
@@ -607,25 +656,44 @@ export class Menu extends Widget {
    * This listener is attached to the menu node.
    */
   private _evtKeyDown(event: KeyboardEvent): void {
-    // A menu handles all keydown events.
-    event.preventDefault();
-    event.stopPropagation();
-
     // Fetch the key code for the event.
     let kc = event.keyCode;
 
-    // Enter
-    if (kc === 13) {
+    // A menu handles all keydown events, except for tab, which we let propagate.
```

Comment:
```suggestion
    // A menu handles all keydown events, except for tab, which we let propagate
    // for accessibility, so that users can navigate the UI using only their keyboard.
```